nrf_modem/doc/at_interface.rst

```diff
@@ -101,13 +134,19 @@ The following snippet shows how to use :c:func:`nrf_modem_at_scanf` to read the
 		}
```

Comment:
Some people may disagree but I'd add example/text of how to deal with the problematic parts of the AT response with scanf:
- Empty string in AT response
- Handling enums
- ",," (comma, comma) meaning empty field

Reply:
It's a good idea, I can collect a few of these examples and show how to handle those in a follow-up PR.
Let's get this in for v1.8.0 in the meantime :)

Reply:
_and show how to handle those in a follow-up PR_

Approving on this promise :)